coopr-ui/app/services/helpers.js

```diff
@@ -34,7 +34,28 @@ angular.module(PKG.name+'.services').value('myHelpers', {
       total += timeObj.minutes * 60;
     }
     return total * 1000;
+  },
+
+  /**
+   * Get human readable string from milliseconds.
+   * @return {String} readable millisecond display.
+   */
+  millisecondsToString: function (milliseconds) {
+    var parsedTime = this.parseMilliseconds(milliseconds);
```

Comment:
not sure we can guarantee the method will be called with the correct context for `this` to work.